# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#result definition 平均值 标准差
float32 mean
float32 std_dev
